server/lib/github.js

```diff
@@ -94,7 +94,9 @@ const getTree = (repository, branch, sha) =>
   new Promise((resolve, reject) => {
     try {
       const github = new GitHubApi({
-        version: '3.0.0'
+        version: '3.0.0',
+        host: config('GITHUB_HOST') || 'api.github.com',
+        pathPrefix: config('GITHUB_API_PATH') || ''
```

Comment:
Is the `/api/v3` fixed for all Github Enterprise deployments? If yes, can't we just add it to the `url` if the `GITHUB_HOST ` is set? So if `GITHUB_HOST` is set _and_ different from `api.github.com`, add the fixed url part `/api/v3` to the `url`.